### ISSUE TITLE
Focus prototype on "Firefox team" mindset

### ISSUE THREAD

**Issue by simonwex** [mozilla/webmaker-addons#53]
- [x] Remove references to Webmaker
- [x] Remove "save as HTML" functionality

**Comment by toolness** [comment]
Done in 1fd1a0579362a9e219128431f6e11b55f40b5805. (I actually only removed the "save as HTML" functionality, but as far as I can tell, this is the only reference to Webmaker.)

FWIW, though, I don't think the anti-Webmaker aspect should necessarily negate the ability to share the creation as **native web content** rather than a static image: one of the things that users can do, for instance, is bring animated GIFs into their creations, which aren't animated when exporting to PNG. Another interesting possibility that requires a web-native export format is mentioned in #36, where clicking on an image could potentially take a user to the page that the image was imported from.

**Comment by xmatthewx** [comment]
Agreed. Remove branding, but preserve the ability to export as html or page.
